defaults/main.yml

```diff
@@ -9,14 +9,15 @@ gitea_home: "/var/lib/gitea"
 gitea_shell: "/bin/false"
 gitea_systemd_cap_net_bind_service: false
 
-gitea_repository_root: "{{ gitea_home }}"
+gitea_repository_root: "{{ gitea_home }}/repos"
```

Comment:
With this change here I am unhappy and don't really know what to say.

Putting the repos into a subfolder makes perfect sense.
But if you just change that for existing installations... Then the repos are stored in the database but the matching data is not found. Until an administrator moves them...

Unfortunately, I don't know how to find a nice solution for this.

Reply:
yes, i know ;( ... lets discuss it in #63

Reply:
if this is the only part - please drop this commit and merge this PR otherwise

Reply:
i drop this commit in this PR ...